ember-source.gemspec

```diff
@@ -11,7 +11,7 @@ Gem::Specification.new do |gem|
   gem.homepage      = "https://github.com/emberjs/ember.js"
   gem.license       = 'MIT'
 
-  gem.version       = Ember.rubygems_version_string
+  gem.version       = Ember::VERSION
 
   gem.files = %w(VERSION) + Dir['dist/*.js', 'lib/ember/*.rb']
```

Comment:
I think this also needs updating?

Reply:
Added `package.json`.
Question, though, do we still need that `VERSION` file?

Reply:
It seems to be in use in the deploy script:

    gem push ember-source-`cat VERSION`.gem